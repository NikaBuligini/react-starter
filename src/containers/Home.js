import React from 'react';
import { Helmet } from 'react-helmet';

const Home = () =>
  <div>
    <Helmet>
      <title>Home - Title</title>
    </Helmet>
    <div className="container">
      <div className="row">
        <div className="col-md-offset-2 col-md-8">Home</div>
      </div>
    </div>
  </div>;

export default Home;
