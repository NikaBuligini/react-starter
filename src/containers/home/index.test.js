import React from 'react';
import renderer from 'react-test-renderer';
import Home from './index';

test('renders Home component', () => {
  const component = renderer.create(<Home />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
