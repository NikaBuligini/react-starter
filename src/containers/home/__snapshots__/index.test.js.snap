// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render the Home page 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="row justify-content-center"
    >
      <div
        className="col-md-8"
      >
        <span>
          Home
        </span>
        <div>
          <div
            className="s16lz4en-0 lhhFDI"
          >
            <div
              className="s16lz4en-1 erqRNC"
              color="#999"
            />
            <div
              className="s16lz4en-1 HGoYe"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 cAmvJp"
              color="#999"
            />
            <div
              className="s16lz4en-1 hscrFX"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 dcprgb"
              color="#999"
            />
            <div
              className="s16lz4en-1 hhQqnw"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 kotsvt"
              color="#999"
            />
            <div
              className="s16lz4en-1 iUcVTz"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 hbaxwP"
              color="#999"
            />
            <div
              className="s16lz4en-1 estqAr"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 iFpkex"
              color="#999"
            />
            <div
              className="s16lz4en-1 dEwtnY"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 eNiwnR"
              color="#999"
            />
            <div
              className="s16lz4en-1 fMXxsh"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 gkCjhH"
              color="#999"
            />
            <div
              className="s16lz4en-1 yzMHk"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 iPkzKV"
              color="#999"
            />
            <div
              className="s16lz4en-1 hzHcdL"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 gWCVfP"
              color="#999"
            />
            <div
              className="s16lz4en-1 lhlDJF"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 kYcIaX"
              color="#999"
            />
            <div
              className="s16lz4en-1 gAFXwd"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 kAwtMg"
              color="#999"
            />
            <div
              className="s16lz4en-1 hfQxCC"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 jpSCQN"
              color="#999"
            />
            <div
              className="s16lz4en-1 laAWtT"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 iCXHAW"
              color="#999"
            />
            <div
              className="s16lz4en-1 cIQHkA"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 bnZdjG"
              color="#999"
            />
            <div
              className="s16lz4en-1 iwOBOE"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 efFHlT"
              color="#999"
            />
            <div
              className="s16lz4en-1 dZxRrp"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 eGoDnb"
              color="#999"
            />
            <div
              className="s16lz4en-1 kXTJLe"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 cmNfke"
              color="#999"
            />
            <div
              className="s16lz4en-1 kpgeni"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 bTSdwK"
              color="#999"
            />
            <div
              className="s16lz4en-1 bLevNw"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 eMoGtR"
              color="#999"
            />
            <div
              className="s16lz4en-1 VktSh"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 idBDSP"
              color="#999"
            />
            <div
              className="s16lz4en-1 cTGDfv"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 kJOdzc"
              color="#999"
            />
            <div
              className="s16lz4en-1 fasPeZ"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 eTNEuQ"
              color="#999"
            />
            <div
              className="s16lz4en-1 ghLzKa"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 ebfKzr"
              color="#999"
            />
            <div
              className="s16lz4en-1 faAfHA"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 iSFBKA"
              color="#999"
            />
            <div
              className="s16lz4en-1 cwYRlp"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 kMfoOm"
              color="#999"
            />
            <div
              className="s16lz4en-1 fXZHvq"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 eCOLUR"
              color="#999"
            />
            <div
              className="s16lz4en-1 dXayHD"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 kXsFwV"
              color="#999"
            />
            <div
              className="s16lz4en-1 eXZpuU"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 gmgQcD"
              color="#999"
            />
            <div
              className="s16lz4en-1 bTJWBf"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 ElPxV"
              color="#999"
            />
            <div
              className="s16lz4en-1 dVZqfq"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 frnMrw"
              color="#999"
            />
            <div
              className="s16lz4en-1 iNsBVm"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 dWSCEs"
              color="#999"
            />
            <div
              className="s16lz4en-1 hzIDBx"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 cEvfxY"
              color="#999"
            />
            <div
              className="s16lz4en-1 jpTCme"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 jzHSdw"
              color="#999"
            />
            <div
              className="s16lz4en-1 gdZybs"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 bPqFwm"
              color="#999"
            />
            <div
              className="s16lz4en-1 dgoZzR"
              color="#fdb37f"
            />
            <div
              className="s16lz4en-1 enIxjA"
              color="#999"
            />
            <div
              className="s16lz4en-1 bZkHUb"
              color="#fdb37f"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
